src/Playlist.js

```diff
@@ -813,7 +815,8 @@ export default class {
                 "isActive": this.isActiveTrack(track),
                 "shouldPlay": this.shouldTrackPlay(track),
                 "soloed": this.soloedTracks.indexOf(track) > -1,
-                "muted": this.mutedTracks.indexOf(track) > -1
+                "muted": this.mutedTracks.indexOf(track) > -1,
+                "waveBackgroundColor": track.waveBackgroundColor
```

Comment:
since this is track specific, don't have to pass this into the data props. Can just use this property directly in the track class like the new customClass property is being accessed.